tier4_planning_msgs/msg/SafetyFactor.msg

```diff
@@ -0,0 +1,28 @@
+# msg format version
+string version_msg_format
+
+# constants for common use
+uint16 UNKNOWN = 0
+
+# constants for factor type
+uint16 POINTCLOUD = 1
+uint16 OBJECT = 2
+
+# variables
+uint16 type
+
+# use only for predicted objects
+unique_identifier_msgs/UUID object_id
+
+# use only for predicted objects
+autoware_perception_msgs/PredictedPath predicted_path
```

Comment:
If you know the UUID, you can refer to the path of the predicted_object topic. Is this to improve debugging convenience? I am concerned about the increase in data size due to the path.